src/Servers/Kestrel/Core/src/Internal/SniOptionsSelector.cs

```diff
@@ -68,7 +70,7 @@ internal sealed class SniOptionsSelector
             {
                 // This might be do blocking IO but it'll resolve the certificate chain up front before any connections are
                 // made to the server
-                sslOptions.ServerCertificateContext = SslStreamCertificateContext.Create((X509Certificate2)sslOptions.ServerCertificate, additionalCertificates: null);
+                sslOptions.ServerCertificateContext = SslStreamCertificateContext.Create((X509Certificate2)sslOptions.ServerCertificate, additionalCertificates: fullChain, offline: fullChain != null);
```

Comment:
Specifying `offline: true` will disable OCSP stapling on Linux.  I recommend just leaving it as online.

Reply:
Okay I don't recall where exactly I got the idea to try setting offline, I can't find it from any of the prior iterations anymore, so I will just remove it